src/pages/Messages/index.js

```diff
@@ -194,20 +226,20 @@ const MessagesPage = () => {
           {contacts.length > 0 ? (
             <p className="messages-section-title">Messages</p>
           ) : null}
-          {contacts.map(contact => {
+          {orderedContacts.map(contact => {
             const contactMessages = messages[contact.pk] ?? [];
             const lastMessage = (() => {
```

Comment:
If you could extract this function too into a separate one defined in the component's root, that would be great! Should make the code again, faster and more readable